src/py-opentimelineio/opentimelineio/adapters/svg.py

```diff
@@ -0,0 +1,1017 @@
+                                                   repr(svg_rect.height),
+                                                   svg_color(fill_color),
+                                                   repr(stroke_width),
+                                                   svg_color(border_color),
+                                                   repr(fill_color[3] / 255.0))
+    lines.append(rect_str)
+
+
+def draw_line(start_point, end_point, stroke_width,
+              stroke_color=COLORS['black'], is_dashed=False):
+    point1 = convert_point_to_svg_coordinates(start_point)
+    point2 = convert_point_to_svg_coordinates(end_point)
+    if is_dashed:
+        line = r'<line x1="{}" y1="{}" x2="{}" y2="{}" ' \
+               r'style="stroke:{};stroke-width:{};opacity:{};' \
+               r'stroke-linecap:butt;' \
+               r'stroke-dasharray:4 1" />'.format(repr(point1.x),
+                                                  repr(point1.y),
+                                                  repr(point2.x),
+                                                  repr(point2.y),
+                                                  svg_color(stroke_color),
+                                                  repr(stroke_width),
+                                                  repr(stroke_color[3]))
+    else:
+        line = r'<line x1="{}" y1="{}" x2="{}" y2="{}" ' \
+               r'style="stroke:{};stroke-width:{};opacity:{};' \
+               r'stroke-linecap:butt;" />'.format(repr(point1.x),
+                                                  repr(point1.y),
+                                                  repr(point2.x), repr(point2.y),
+                                                  svg_color(stroke_color),
+                                                  repr(stroke_width),
+                                                  repr(stroke_color[3]))
+    lines.append(line)
+
+
+def draw_arrow(start_point, end_point, stroke_width, stroke_color=COLORS['black']):
+    point1 = convert_point_to_svg_coordinates(start_point)
+    point2 = convert_point_to_svg_coordinates(end_point)
+    direction = Point(point2.x - point1.x, point2.y - point1.y)
+    direction_magnitude = math.sqrt(direction.x * direction.x +
+                                    direction.y * direction.y)
+    inv_magnitude = 1.0 / direction_magnitude
+    direction = Point(direction.x * inv_magnitude, direction.y * inv_magnitude)
+    point2 = Point(point2.x - 9.0 * direction.x, point2.y - 9.0 * direction.y)
+    triangle_tip = Point(point2.x + 9.0 * direction.x,
+                         point2.y + 9.0 * direction.y)
+    perpendicular_dir = Point(-direction.y, direction.x)
+    triangle_pt_1 = Point(point2.x + 4.5 * perpendicular_dir.x,
+                          point2.y + 4.5 * perpendicular_dir.y)
+    triangle_pt_2 = Point(point2.x - 4.5 * perpendicular_dir.x,
+                          point2.y - 4.5 * perpendicular_dir.y)
```

Comment:
Throughout this code there are a lot of ["magic number"](https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants) usages here (`9.0` and `4.5`). This should be avoided.

Reply:
I think I'd chosen these numbers according to what worked well at the time. Would moving these to variables with appropriate names be the correct way?

Reply:
Yeah, mostly it's about just giving names for the values. If someone needs to tune them later they'll be unaware of what they mean.
It may be something like (I'm actually not sure if these are the right names):
```python
arrow_height = 9.0
arrow_width = 4.5
```

Reply:
name it arrowhead_length or similar? and use arrowhead_length * 0.5 as appropriate.